L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker_cfg.py

```diff
@@ -86,7 +78,7 @@
 
 
 # DTC emulation
-process.load('L1Trigger.TrackerDTC.ProducerES_cff')
+#process.load('L1Trigger.TrackerDTC.ProducerES_cff')
```

Comment:
Is this line necessary? Could it be removed?

Reply:
done